src/EventListener/DeviceViewListener.php

```diff
@@ -66,7 +66,7 @@ public function onKernelResponse(FilterResponseEvent $event): void
                 0,
                 '/',
                 null,
-                true,
+                $request->isSecure(),
                 true,
                 false,
                 Cookie::SAMESITE_STRICT
```

Comment:
@arunkarnati do you want to do samesite none when not secure or whatever symfony default is?

Reply:
@gdbrown - I don't think a change to `samesite` attribute is necessary here since the `device_type` cookie can stay within the website boundaries. Thoughts?

Reply:
@arunkarnati samesite only works when secure is also set.  so i'd just do something like ` $request->isSecure() ? Cookie::SAMESITE_STRICT : whateverthesymfonydefaultis`

Reply:
here's what this https://web.dev/samesite-cookies-explained/ article says: 

> Cookies with SameSite=None must also specify Secure, meaning they require a secure context.

In our case, we are setting the cookie to `SameSite=Strict`, which should work regardless of the secure attribute.